build.sbt

```diff
@@ -69,3 +69,8 @@ lazy val it =
   (project in file("it"))
     .enablePlugins(PlayScala)
     .dependsOn(microservice % "test->test")
+
+
+addCommandAlias("coverageCheck", ";clean;coverage;test;it/test;coverageReport")
+addCommandAlias("prePR", ";scalafmt;test:scalafmt;coverageCheck")
+addCommandAlias("preMerge", ";scalafmtCheckAll;coverageCheck")
```

Comment:
if the scalafmtCheckAll fails will the tests still run and pass? If so it might be difficult to see that scalafmt has failed (similar for the one above, but it is less likely for those to fail as they are doping the actual formatting)

Reply:
Yep, so I've just tested there and if the check fails the command will stop which is nice